code/ui/QuickShop.scss

```
@@ -34,49 +33,84 @@ QuickShop {
         border-top-right-radius: 7px;
 
         Label {
-            &.title, &.credits {
+            &.title,
+            &.credits {
                 font-size: 20px;
             }
+            &.dollarsign {
+                margin-top: 4px;
+            }
         }
     }
 
     Content {
+        width: 100%;
         max-height: 400px;
-        padding: 10px;
+        padding: 16px;
         text-align: center;
-        justify-content: center;
 
         Panel.wrapper {
+            width: 100%;
+            flex-wrap: wrap;
+            justify-content: flex-start;
+
             ItemPanel {
                 flex-direction: column;
                 margin: 10px;
+                // margin-right: 50px;
+                padding-right: 13px;
+
+                .priceholder {
+                    display: flex;
+                    justify-content: center;
+                    margin-top: 10px;
+
+                    Label.price {
+                        font-weight: 400;
+                        font-size: 16px;
+                        margin-right: 1px;
+                    }
 
-                Label.price {
-                    font-weight: 400;
-                    font-size: 16px;
+                    .dollarsign {
```

```suggestion
                    Label.dollarsign {
```
Or wrap it together with the above one :)